### PR TITLE
AP_Camera - use gps singleton

### PR DESCRIPTION
**Pull request by peterbarker**
Requires #7209 to go in.

### CODE REVIEW

File: libraries/AP_Camera/AP_Camera.cpp [hunk 1]
```diff
@@ -245,7 +245,7 @@ void AP_Camera::send_feedback(mavlink_channel_t chan)
     }
 
     mavlink_msg_camera_feedback_send(chan, 
-        gps.time_epoch_usec(),
+                                     AP::gps().time_epoch_usec(),
```
Comment (amilcarlucas) [inline libraries/AP_Camera/AP_Camera.cpp]: Why all the whitespace ?

Comment (peterbarker) [reply libraries/AP_Camera/AP_Camera.cpp]: @amilcarlucas Lines it up correctly with the argument list!

Small improvement ;-)

Comment (OXINARF) [reply libraries/AP_Camera/AP_Camera.cpp]: I would say it is a great improvement! But doing this line and leaving the next ones in the same indentation leaves it in a strange (and worse) state.

Comment (peterbarker) [reply libraries/AP_Camera/AP_Camera.cpp]: OK, I found a third way and moved the `chan` argument down :-)

**Comment by OXINARF** [comment]
Merged, thanks!

**Pull request merged**